### DISCUSSION TITLE
New firefly & shimmer wallet update - all tokens gone.

### DISCUSSION THREAD

**Discussion by baz59** [iotaledger/firefly#8029] [Help & Support]
Hi I just updated both wallets , the wallets had a claim lottery in them. When the wallets updated the Tokens have gone - exported out - Now showing 0 SMR & .02 iota. I think I have been scammed both wallets drained when updating. This is a problem as how does an updated version allow this to happen.

**Comment by begonaalvarezd** [comment]
We cant prevent users from being sent NFTs, in this case this was a scam NFT and unfortunately we cant do anything about it, the tangle is public and anybody can pick addresses from the node's indexer api and send assets to them. You should know that you can never trust any third party service and you should never share your seed to anyone nor enter it in any random website. 

If you did not enter your mnemonic to any random website (the website of the scam in this case) you are safe, you can simply ignore the scam NFT or burn it to get the storage deposit of it. If you upgraded firefly from v1.x to v2.x you need to run the balance finder to let firefly re-descover your addresses, you can do so from settings > advanced > balance finder. Additionally, if you didnt claim your shimmers--> quick information about SMR: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app because your rewards are in the shimmer network, not the iota network. You must download the firefly shimmer app and claim them, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/
